packages/orchestration/test/facade.test.ts

```diff
@@ -79,4 +80,51 @@ test('chain info', async t => {
   t.deepEqual(await vt.when(result.getChainInfo()), mockChainInfo);
 });
 
+test.serial('faulty chain info', async t => {
+  const { facadeServices, commonPrivateArgs } = await commonSetup(t);
+
+  // XXX relax again so setupZCFTest can run. This is also why the tests are serial.
+  reincarnate({ relaxDurabilityRules: true });
+  const { zcf } = await setupZCFTest();
+
+  // After setupZCFTest because this disables relaxDurabilityRules
+  // which breaks Zoe test setup's fakeVatAdmin
+  const zone = provideDurableZone('test');
+  const vt = prepareSwingsetVowTools(zone);
+
+  const orchKit = provideOrchestration(
+    zcf,
+    zone.mapStore('test'),
+    {
+      agoricNames: facadeServices.agoricNames,
+      timerService: facadeServices.timerService,
+      storageNode: commonPrivateArgs.storageNode,
+      orchestrationService: facadeServices.orchestrationService,
+      localchain: facadeServices.localchain,
+    },
+    commonPrivateArgs.marshaller,
+  );
+
+  const { chainHub, orchestrate } = orchKit;
+
+  const { stakingTokens, ...sansStakingTokens } = mockChainInfo;
+
+  chainHub.registerChain('mock', sansStakingTokens);
+  chainHub.registerConnection(
+    'agoric-3',
+    mockChainInfo.chainId,
+    mockChainConnection,
+  );
+
+  const handle = orchestrate('mock', {}, async orc => {
+    const chain = await orc.getChain('mock');
+    const account = await chain.makeAccount();
+    return account;
+  });
+
+  await t.throwsAsync(handle(), {
```

Comment:
```suggestion
  await t.throwsAsync(when(handle()), {
```

Reply:
`orchestrate` already does that: https://github.com/Agoric/agoric-sdk/blob/6e8430b3eee0cc7efda52c32b2aef3fa78e6396b/packages/orchestration/src/facade.js#L89-L93

so when adding `vt.when` here, the exception is still unhandled:
```
SES_UNHANDLED_REJECTION: (Error#1)
Error#1: chain info lacks staking denom
  at makeError (file:///opt/agoric/agoric-sdk/node_modules/ses/src/error/assert.js:347:61)
```